backend/package.json

```diff
@@ -1,16 +1,26 @@
 {
   "name": "backend",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": "Node.js + MongoDB: JWT Authentication & Authorization",
+  "main": "server.js",
   "scripts": {
```

Comment:
We might need to add a start script here that would be used to start the server. In this case it would be something like `start: "node server.js"` that would allow us run `npm start` to start the server.